src/commands/config.js

```diff
@@ -365,12 +385,36 @@ exports.handler = async ({func} = {}) => {
     }
   };
 
-  // get values from user
-  const newConfig = await inquirer.prompt(prompts);
+  let newConfig = defaultConfig;
+  let nonInteractive = false;
+
+  const overrideFromArgument = (key, value) => {
+    if (value) {
```

Comment:
I'd instead do early return when value is false or length is zero, just to make it easier to read.

Reply:
I learned to do as less returns as possible and doing so is not that bad i think...
You would like `if(!value) {return;}` ?

Reply:
Removed the function completely

Reply:
So, maybe this suits you :D